index.js

```diff
@@ -50,6 +51,11 @@ module.exports = (source, destination, options = {}) => {
 			throw new CpyError(`Cannot glob \`${source}\`: ${error.message}`, error);
 		}
 
+		if (options.filter !== undefined) {
+			const filteredFiles = await pFilter(files, options.filter, {concurrency: 1024});
```

Comment:
Why 1024? Why set it at all?

Reply:
With 1024 concurrency performance should never degrade(except in some extremely narrow cases like say getting response by network on a really fast server), and numbers of files below 1024 won't be affected at all, but in case of big amount of files and async filters even relatively simple filters might start holding a lot of memory(so, memory spikes and reduced performance), this limit will guard against that.

So, as a sane default. Not noticeable in absolute majority of cases, but increasing performance in some subset.

Reply:
Actually, whole `cpy` should probably have some concurrency(possibly configurable) set, it will speed things up(in majority of cases with big number of files)